First Task/src/main/java/task/Functional.java

```diff
@@ -0,0 +1,23 @@
+package task;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+class Functional {
+    static int sum(List<Integer> list) {
+        int sum = 0;
+        for (Integer elem : list)
+            sum += elem;
```

Comment:
Everything is correct, but it is better to use curly braces around the body or I also accept writing it without braces if it can be written in one line, for instance:
```java
for (Integer elem : list) sum += elem;
```
And below you can see an example for body enclosed in braces.